app/models/overview.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# At-a-glance information for the POD aggregator and a single organization
+class Overview
```

Comment:
Any thoughts on this class name? `Overview` seems a bit generic but we already use Dashboard for something else in the app. Kind of seems like a `UserDashboard` to me. Maybe we can deal with this later. Just thinking...